src/app/utils/url-generator.js

```diff
@@ -45,11 +45,13 @@ export const embedUrl = (sandbox: Sandbox) => {
 };
 
 export const frameUrl = (append: string = '') => {
+  const path = append.indexOf('/') === 0 ? append.substr(1) : append;
```

Comment:
Yes perhaps it could be more consistently handled by not getting that initial slash in the first place. But it does and I figured we are better safe than sorry. It's the slash the user enters in the initial path field. So if u want I can strip it earlier.